rules/DeadCode/NodeAnalyzer/IsClassMethodUsedAnalyzer.php

```diff
@@ -112,13 +112,14 @@ private function isClassMethodCalledInLocalArrayCall(Class_ $class, ClassMethod
     {
         /** @var Array_[] $arrays */
         $arrays = $this->betterNodeFinder->findInstanceOf($class, Array_::class);
+        $classMethodName = $this->nodeNameResolver->getName($classMethod);
 
         foreach ($arrays as $array) {
             if ($this->isInArrayMap($class, $array)) {
                 return true;
             }
 
-            $arrayCallable = $this->arrayCallableMethodMatcher->match($array, $scope);
+            $arrayCallable = $this->arrayCallableMethodMatcher->match($array, $scope, $classMethodName);
```

Comment:
(note)
If it use `$secondItemValue` instead of `$classMethodName`, then cause regression below.

```
There were 2 failures:

1) Rector\Tests\CodeQuality\Rector\Array_\CallableThisArrayToAnonymousFunctionRector\CallableThisArrayToAnonymousFunctionRectorTest::test with data set #20 ('...')
Failed on fixture file "skip_other_class_has_construct_parameter.php.inc"
Failed asserting that string matches format description.
--- Expected
+++ Actual
@@ @@
 {
     public function run()
     {
-        return [OtherClass::class, 'someMethod'];
+        return function () {
+            return (new \Rector\Tests\CodeQuality\Rector\Array_\CallableThisArrayToAnonymousFunctionRector\Source\OtherClass())->someMethod();
+        };
     }
 }
 ?>

2) Rector\Tests\CodeQuality\Rector\Array_\CallableThisArrayToAnonymousFunctionRector\CallableThisArrayToAnonymousFunctionRectorTest::test with data set #21 ('...')
Failed on fixture file "skip_same_class_has_construct_parameter.php.inc"
Failed asserting that string matches format description.
--- Expected
+++ Actual
@@ @@

 public function run()
 {
- return [SkipSameClassHasConstructParameter::class, 'someMethod'];
+ return function () {
+ return (new \Rector\Tests\CodeQuality\Rector\Array_\CallableThisArrayToAnonymousFunctionRector\Fixture\SkipSameClassHasConstructParameter())->someMethod();
+ };
 }

 public function someMethod()
```